cypress/integration/ete/register_email_sent.spec.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/// <reference types="cypress" />
+import { injectAndCheckAxe } from '../../support/cypress-axe';
+
+describe('Registration email sent page', () => {
+  context('A11y checks', () => {
+    it('has no detectable a11y violations on the registration email sent page', () => {
+      // set encrypted email
+      cy.setCookie('PLAY_SESSION_2', testUserEncryptedEmailCookie, {
+        log: true,
+      });
+      cy.visit(`/register/email-sent`);
+      injectAndCheckAxe();
+    });
+  });
+
+  const testUserEmail = 'test+13411@guardian.com';
+  const testUserEncryptedEmailCookie = Buffer.from(
+    JSON.stringify(testUserEmail),
+  ).toString('base64');
+
+  it('should load the page with a success banner given a valid encrypted email cookie', () => {
+    // set encrypted email
+    cy.setCookie('PLAY_SESSION_2', testUserEncryptedEmailCookie, {
+      log: true,
+    });
+    cy.visit(`/register/email-sent`);
+    cy.contains('Email sent');
+    cy.contains(testUserEmail);
+    cy.contains('Resend email');
+    cy.contains('Change email address');
+  });
+
+  // This test depends on this Mailslurp account already being sent to the /guest endpoint.
+  const resendTestEmail = {
+    email: '7b4f97c5-79cd-4a3c-9212-cfb6db959dfc@mailslurp.com',
+    inbox: '7b4f97c5-79cd-4a3c-9212-cfb6db959dfc',
+  };
+  // Set encrypted email using test email defined above.
+  const testMailSlurpEncryptedEmailCookie = Buffer.from(
```

Comment:
It might be nice to name this something more descriptive, like `encryptedEmail`?

Reply:
how about `mailSlurpEncryptedEmail` and `userEncryptedEmail` for the two emails we have in the file?

Reply:
Ha, it's so true that naming things is hard It is very much worth putting time in on it though - our future selves will thank us!

So in this file we have

```
testUserEmail
testUserEncryptedEmailCookie
resendTestEmail
testMailSlurpEncryptedEmailCookie
```

But these names don't help with the understanding of what the tests are doing. I think mainly because they have such long names but also because I'm not sure if I need to think differently about them based on their names

Breaking this down, there are only two tests using these variables. 

1. `should load the page with a success banner given a valid encrypted email cookie`
and
2. `should resend the email when the resend button is clicked`

1 uses `testUserEmail` & `testUserEncryptedEmailCookie`
2 uses `resendTestEmail` & `testMailSlurpEncryptedEmailCookie`

So I can now see that we have 2 email addresses and two encrypted strings used when setting a cookie.

But maybe we can use the same email address in both tests? I don't see any data being mutated and I think the resend logic works the same in both situations?

Like that we could simply have `email` and `encryptedEmail` which represents what the strings are and all the other context can drop away? Of course, that only works if these string really do work fine when you use them in both tests

Reply:
Yeah I think that's a really good perspective on the problem. I think you're 100% right, we don't need two email addresses here. I will change so that we only use the mailslurp email. No data is mutated and it should work fine when we use it for both tests (the other tests just want to find a valid email on the page)